src/views/tutorial-view/utils/mdx-components.tsx

```diff
@@ -43,6 +46,13 @@ const MDX_COMPONENTS = {
   InteractiveLabCallout,
   img: Image,
   VideoEmbed,
+  h1: (props) => makeHeadingElement(1, props),
+  h2: (props) => makeHeadingElement(2, props),
+  h3: (props) => makeHeadingElement(3, props),
+  h4: (props) => makeHeadingElement(4, props),
+  h5: (props) => makeHeadingElement(5, props),
+  h6: (props) => makeHeadingElement(6, props),
+  p: (props) => <Text {...props} className={devDotStyles.p} />,
```

Comment:
Just copied the instantiation pattern used in docs here to be quick about it. There's room to improve this generation I think but it's a quick fix for now.